pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+minversion = 6.0
+testpaths=
+    tests
+
```

Comment:
Extra empty line.

Reply:
Also in other files.

Reply:
I removed the extra blank lines at the end of all respective files